packages/legacy-workbench/src/js/angular/guides/steps/complex/resources/plugin.js

```diff
@@ -27,16 +146,15 @@ PluginRegistry.add('guide.step', [
                             }
                             const previousStep = services.ShepherdService.getPreviousStepFromHistory(stepId);
                             return previousStep.options.initPreviousStep(services, previousStep.options.id);
-                        }
-                    }, options)
-                }, {
-                    guideBlockName: 'read-only-element',
-                    options: angular.extend({}, {
+                        },
+                        ...options
+                    }
+                },
+                {
+                    guideBlockName: 'resource-results-explain',
+                    options: {
                         content: 'guide.step_plugin.table-graph-overview',
-                        scrollToHandler: GuideUtils.scrollToTop,
-                        elementSelector: GuideUtils.CSS_SELECTORS.SPARQL_RESULTS_ROWS_SELECTOR,
-                        class: 'table-graph-overview',
-                        placement: 'top',
+                        skipUrl: true,
                         beforeShowPromise: () => GuideUtils.waitFor(`.resource-info a.source-link[href="${options.iri}"]`, 3)
```

Comment:
Why is this not extracted with the step?

Reply:
This can’t be moved to the extracted step because it is a specific case for the complex step. It is not relevant for other usages.